collection-scripts/gather

```diff
@@ -19,7 +19,7 @@ resources+=(ns/default ns/openshift ns/kube-system)
 resources+=(storageclasses persistentvolumes volumeattachments)
 
 # Networking Resources
-resources+=(clusternetworks hostsubnets)
+resources+=(network.config.openshift.io networkoperator.openshift.io clusternetworks hostsubnets)
```

Comment:
I believe the networkoperator is already captured by [clusteroperators on line 7](https://github.com/openshift/must-gather/pull/94/files#diff-49627f729628fe028600d501df843ea1R7)

I believe network.config.openshift.io is already captured because of the operator<>config mapping (with the networkoperator)

Reply:
> I believe the networkoperator is already captured by [clusteroperators on line 7](https://github.com/openshift/must-gather/pull/94/files#diff-49627f729628fe028600d501df843ea1R7)

it should be.  If it isn't, the related resources on the clusteroperator resource should be updated to reflect the need.

Reply:
Will this fail if clusternetworks and hostsubnets do not exist? They're only present for openshift-sdn. Other networking providers won't have the resources defined.

Reply:
There's no need to collect the clusternetwork object. Its contents are completely deterministic given `network.operator.openshift.io` (which is missing a `.` here btw).

> Will this fail if clusternetworks and hostsubnets do not exist? They're only present for openshift-sdn. Other networking providers won't have the resources defined.

This is a problem with most of `gather_network_logs` as well. Probably we should be plugin-agnostic in `gather`, and check what the configured network plugin is in `gather_network_logs`, and collect accordingly.

Reply:
Can / should we add a dpendency in the must-gather binary for the networking operator and the denoted objects here? 

@deads2k